danger.c

```diff
@@ -0,0 +1,10 @@
+# crosstest
+   char src[40];
++
+# I like banned API methods
++gets(src); 
```

Comment:
[Banned C function detected (gets)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS181021.md#DS181021)
gets will read in as many bytes as are provided, regardless of the size of buffer recieving the bytes.  This allows for conditions that cause a buffer overlow in the destination buffer.

*Fix Guidance:* Use gets_s or fgets if possible. fgets has different behavior from gets (may include newline for example), so care should be taken if it is used.


Suggestions:
[1] +gets_s(src, <size of src>);
[2] +fgets(src, <size of src>, stdin);